gpu: true
fps_filter_ratio: 0.9